### DISCUSSION TITLE
Acemi hırsız kim

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#112] [Zeka Sorusu]
Acemi bir hırsız, bilmeden bir emniyet müdürünün arabasını çalar. Polis hızlı bir şekilde olayı araştırmaya başlar ve olay sırasında civarda görülen tüm şüpheli şahıslar gözaltına alır. Emniyet müdürü, olayı çok ciddiye aldığından, hem sorgulamayı bizzat yapar hem de bir yalan makinesi kullanır. Her şüpheli, sorgu sırasında üçer Şüpheli ifade verirler.

```
Şüpheli A:
1. Ben lisedeyken C ile aynı sınıftaydım.
2. B isimli şüphelinin ehliyeti yoktur.
3. Hırsız çaldığı arabanın emniyet müdürüne ait olduğunu bilmiyordu.
Şüpheli B:
1.Arabayı C çaldı.
2.A, suçlu değildir.
3.Ben hayatım boyunca bir arabanın arkasında bile durmadım.
Şüpheli C:
1.A’yı hiç tanımam; hiçbir yerde de görmedim.
2. B masumdur.
3.D suçludur.
Şüpheli D:
1.C masumdur.
2.Ben yapmadım.
3.A suçludur.
```

Bu kadar çok çelişkili ifadenin ardından emniyet müdürünün kafası iyice karışır. Durumu daha da kötü yapan şey ise yalan makinesinin de çalışması gerektiği gibi çalışmadığının anlaşılmasıdır. Yalan makinesinin verdiği rapor, hırsızın kimliği ile ilgili ifadelerden sadece dördünün kesinlikle doğru olduğunu belirtiyordu (Hangilerinin olduğunu değil ) Bu durumda hırsız kimdir?

**Comment by MnG-Official** [comment]
Şüpheli D olabilir acemi hırsız. Sonuca ulaşmak için önce ifadelerin analizi yapmalıyız:
*Şüpheli A ile C aynı sınıftaydılar ancak C, A yı daha önce görmediğini söylüyor ve bu ifadeler çelişiyor.
*B arabayı C nin çaldığını söylüyor fakat bu ifade A nın ifadesi ile çelişiyor.
*C, B nin masum olduğunu söylüyor fakat bu da B nin kendini savunma biçimiyle çelişki içinde.
*D, C masumdur diyor ama C nin ifadesine ters düşmekte söyledikleri.
Biz burada bir de yalan makinesinin sadece 4 ifadeyi doğru söylediği biliyoruz o yüzden kimin hırsız olduğunu öğrenmeden önce bu ifadeleri toplamalı ve doğru olan 4 cümle haline getirmeliyiz(elbette içinde biraz tahminim de var ama o kadarı da olsun yani ve ayrıca ben benim hislerimin yanıltmayacağına inanıyorum.):
*A nın şu ifadesi doğru, onlar C ile aynı sınıftaydılar.
*B nin de şu ifadesi doğru A suçlu değil.
*C nin doğru olan ifadesi ise B nin masum olmasıdır
*D nin ifadesinde ise C nin masum olması doğrudur.
Bunları da son kez toparlarsak:
*Şüpheli A, C ile aynı sınıftaydı ve Şüpheli B, A’nın suçlu olmadığını doğruluyor.
*Şüpheli C, B’nin masum olduğunu söylüyor ve Şüpheli D, C’nin masum olduğunu doğruluyor.
O zaman geriye bir tek Şüpheli D kalıyor ve bu durumda da polis arabasını çalan acemi hırsız Şüpheli D dir. Tabii keşke polis biraz daha ipucu toplasa daha iyi olur.

**Comment by Sevvalgocgun1** [comment]
Hırsız Şüpheli D'dir.
